infrastructure/quick-deploy/aws/armonik.tf

```diff
@@ -76,4 +78,6 @@ module "armonik" {
     image = local.ecr_images["${var.pod_deletion_cost.image}:${try(coalesce(var.pod_deletion_cost.tag), "")}"].image
     tag   = local.ecr_images["${var.pod_deletion_cost.image}:${try(coalesce(var.pod_deletion_cost.tag), "")}"].tag
   })
+
+  depends_on = [module.sqs_service_account]
```

Comment:
Since this module is optional, how is this dependency managed when the module is not deployed ?

Reply:
Wouldn't it be okay because the module is still deployed (just with count 0).. It's the same thing as the dependency you did with both mongodb and mongodb_sharded no? (We pass the two in even though we only deploy one of them)